themes/default/content/templates/kubernetes/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Kubernetes Templates"
+layout: overview
+desc: Pulumi program templates are the fastest way to deploy Kubernetes on AWS, Azure, or Google Cloud Platform. Templates come with predefined infrastructure as code so you can get started instantly.
+meta_desc: Pulumi program templates that make it easy to deploy Kubernetes on AWS, Azure, or Google Cloud Platform.
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# Adjust this value to ensure the new template sorts as you'd like it to sort in the list.
+weight: 99
+---
+
+### What is Kubernetes?
+
+[Kubernetes (k8s)](/kubernetes) is an open source container management and orchestration project. It enables infrastructure engineers and platform teams to deploy and run container services at scale with a control plane that schedules, deploys, and scales containers throughout their lifecycle using a desired state model. The benefits of using k8s include its portability across multiple environments and clouds, high scalability, and extensibility via an ecosystem of extensions and plugins.
+
+**On [AWS](/aws),** you can use Kubernetes with Amazon Elastic Kubernetes Service (EKS).
+
+**On [Azure](/azure),** you can use Kubernetes with Azure Kubernetes Service (AKS).
+
+**On [Google Cloud Platform](/gcp),** you can use Kubernetes with Google Kubernetes Engine (GKE).
+
+### Building and deploying Kubernetes clusters on AWS, Azure, and Google Cloud
+
+[Infrastructure as code](/what-is/what-is-infrastructure-as-code) is an efficient and repeatable way of building Kubernetes clusters and their infrastructure with programming languages and deploying them to your preferred cloud.
+
+Pulumi’s open source, infrastructure as code SDK lets you build and deploy Kubernetes and cloud infrastructure with TypeScript/JavaScript, Python, Go, Java, .NET, and YAML. The main benefits include:
+
+* **Programming Languages:** Define infrastructure as code in your favorite language instead of domain-specific languages or clicking through cloud consoles.
+
+* **Cloud Native Ecosystem Support:** Orchestrate every component (clusters, app workloads, Helm charts, infrastructure, etc.) with code.
```

Comment:
Is there stuff in the parenthesis that could be more K8s specific? @scottslowe

Reply:
@aaronkao @gthuang IMO the focus here needs to be on other cloud native projects/efforts. You could potentially reword it like this (to replace everything after the heading):
> Use Helm charts or support for native Kubernetes API objects, including integrated Kustomize support, to deploy workloads and applications to your new Kubernetes cluster

Reply:
just pushed an update with some tweaks

Reply:
LGTM